docs/running-node-manual/on.md

```diff
@@ -0,0 +1,79 @@
+# Running an Observer Node manually
+
+## Prerequisites
+Make sure you have all [prerequisites](./prerequisites.md) set up
+## Deployment steps
+
+#### 1. Put `cosmovisor` binary to `/usr/bin/`, set proper owner and execution permissions.
+
+#### 2. Locate the genesis app version to genesis application version directory:
+- Create `$HOME/.dcl/cosmovisor/genesis/bin` directory.
+- Copy `dcld` binary to it, set proper owner and execution permissions.
+    Please note that execution permissions on `dcld` should be granted to all (i.e. User, Group and Others classes)
+    because cosmovisor requires execution permission on the application binary to be granted to Others class.
+
+#### 3. Configure CLI:
+- `./dcld config chain-id testnet`
```

Comment:
Please make the <chain-id> param generic and provide an example for the Test Net (should be applied to all docs in this folder).